slidez-fe-core/src/App.tsx

```diff
@@ -14,4 +18,21 @@ function App() {
   )
 }
 
+function App() {
+  return (
+    <div>
+      <HashRouter>
+        <Switch>
+          <PublicRoute
+            exact
+            path={[AppRoute.LOGIN, AppRoute.REGISTRATION]}
+            component={SignPage}
+          />
+          <Route path={AppRoute.ANY} exact component={DefaultApp} />
+        </Switch>
+      </HashRouter>
+    </div>
+  )
+}
```

Comment:
And I'd rather move routing to a separate component, as here may be some more data